packages/shared/src/messages/MessageProxy.ts

```diff
@@ -78,12 +78,14 @@ export class RpcExtension {
           body: result,
           status: 'success',
         } as IMessageResponse);
-      } catch (e) {
+      } catch (err: unknown) {
+        console.error(`Something went wrong on channel ${message.channel}`, err);
```

Comment:
Not sure we should log as the code is generic and having an error may be on the normal path

Reply:
Yes, I removed it